thunderbird/MozillaThunderbird.spec

```diff
@@ -407,22 +409,21 @@ ac_add_options --with-arch=armv7-a
 ac_add_options --disable-webrtc
 %endif
 # mitigation/workaround for bmo#1512162
-%ifarch ppc64le s390x
+%ifarch s390x
 ac_add_options --enable-optimize="-O1"
 %endif
 %ifarch x86_64
 # LTO needs newer toolchain stack only (at least GCC 8.2.1 (r268506)
 %if 0%{?suse_version} > 1500
 ac_add_options --enable-lto
-ac_add_options MOZ_PGO=1
```

Comment:
PGO for is interesting for performance and therefore a good thing for Firefox. For Thunderbird this will kill reproducible builds and therefore propose to not have this here.